lib/publisher/apppublisher.go

```diff
@@ -0,0 +1,40 @@
+package publisher
+
+type RemoteAppType string
+
+const (
+	Webapp     RemoteAppType = "REMOTE_WEB_APP"
+	Server     RemoteAppType = "REMOTE_SERVER"
+	Database   RemoteAppType = "REMOTE_DATABASE"
+	Desktop    RemoteAppType = "REMOTE_DESKTOP"
+	Kubernetes RemoteAppType = "REMOTE_KUBERNETES"
```

Comment:
I need to create these constants on the Appmanagement side as well. But since we are not adding/removing any databases, desktops, and Kubernetes clusters it should be ok

Reply:
I have added code for kind of apps.

Reply:
On the appManagement side have you added them? In AppType.java? Perfect. Thanks. I just did not see that review somehow.